commcare_connect/users/helpers.py

```diff
@@ -32,3 +35,12 @@ def create_hq_user(user, domain, api_key):
     if hq_request.status_code == 201:
         return True
     return False
+
+
+def invite_user(user, opportunity_access):
+    invite_id = uuid4()
+    opportunity_access.invite_id = invite_id
+    opportunity_access.save()
+    url = "https://connect.dimagi.com/opportunity/accept_invite/{invite_id}"
```

Comment:
This shouldn't hard code the base URL. e.g.
```suggestion
    location = reverse("opportunity:accept_invite", args=(invite_id,))
    url = request.build_absolute_uri(location)
```

edit: I think perhaps you intended to use `settings.BASE_ADDRESS` but I don't think that's necessary since you can use the `request` object or there is a utility in `allauth`: `allauth.utils.build_absolute_uri` which pulls the domain from the Site object.

Reply:
nice. i didn't realize `allauth` could do this, or basic `request`. I looked briefly for this functionality and didn't see anything so figured I could just add a setting, but this is much better. Not sure how I missed it.